src/gentropy/dataset/study_index.py

```diff
@@ -47,6 +47,8 @@ class StudyIndex(Dataset):
     A study index dataset captures all the metadata for all studies including GWAS and Molecular QTL.
     """
 
+    QC_FLAG_COLUMN_NAME = "qualityControls"
```

Comment:
I'm not 100% sure about this abstraction. Although i see some value in not hard-coding column names in the business logic, however why only this column and not other? Should this be generalised in all datasets and all column names?

Reply:
reverted